src/guide/migration/key-attribute.md

```diff
@@ -79,8 +79,8 @@ Similarly, when using `<template v-for>` with a child that uses `v-if`, the `key
 ```html
 <!-- Vue 2.x -->
 <template v-for="item in list">
-  <div v-if="item.isVisible" :key="item.id">...</div>
-  <span v-else :key="item.id">...</span>
+  <div v-if="item.isVisible" :key="'heading-' + item.id">...</div>
+  <span v-else :key="'content-' + item.id">...</span>
```

Comment:
I'm not sure about changing this second example. Using a 'duplicate' key with `v-if`/`v-else` was allowed in Vue 2 and I think this example is easier to follow without the string concatenation.